examples/dt/uni05epsilon/README.md

```diff
@@ -0,0 +1,113 @@
+executed against the below specified topology of Red Hat OpenStack Services
+on OpenShift. It also contains a collection of custom resources (CRs) for
+deploying the test environment.
+
+
+## Purpose
+
+This topology is used for executing integration tests that evaluate the
+`default` backends of the below mentioned services.
+
+
+## Environment
+
+
+### Nodes
+
+| Role              | Machine Type | Count |
+| ----------------- | ------------ | ----- |
+| Compact OpenShift | vm           |   3   |
+| OpenStack Compute | vm           |   3   |
+
+
+### Networks
+
+| Name         | Type     | Interface | CIDR            |
+| ------------ | -------- | --------- | --------------- |
+| Provisioning | untagged | nic1      | 172.22.0.0/24   |
+| Machine      | untagged | nic2      | 192.168.32.0/20 |
+| RH OSP       | trunk    | nic3      |                 |
+
+
+#### VLAN networks in RH OSP
+
+| Name        | Type        | CIDR              |
+| ----------- | ----------- | ----------------- |
+| ctlplane    | untagged    | 192.168.122.0/24  |
+| internalapi | VLAN tagged | 172.17.0.0/24     |
+| storage     | VLAN tagged | 172.18.0.0/24     |
+| tenant      | VLAN tagged | 172.19.0.0/24     |
+| external    | VLAN tagged | 192.168.32.0/20   |
+
+
+### Services, enabled features and configurations
+
+| Service          | configuration           | Lock-in coverage?  |
+| ---------------- | ----------------------- | ------------------ |
+| Barbican         | (default)               | Must have          |
+| Cinder           | iSCSI/netapp, NFS, ceph | Must have          |
+| Cinder Backup    | Swift                   | Must have          |
+| Glance           | cinder/NFS, ceph, swift | Must have          |
+| Horizon          | N/A                     | Must have          |
+| Ironic           |                         | Must have          |
+| Neutron          | Geneve (OVN)            | Must have          |
+| Octavia          | act-stby                | Must have          |
+| Swift            | (default)               | Must have          |
+
+
+#### Support services
+
+The following table lists services which are not the main focus of the testing
+(which may be covered by additional scenarios), but are required for the DT to
+work properly and can be deployed with any/default configuration.
+
+| Service          | Reason                     |
+| ---------------- |--------------------------- |
+| Nova             | needed by scenario testing |
+| Keystone         | needed by all services     |
+| Heat             | needed by Telemetry        |
+| Redis            | needed by Octavia          |
+
+
+### Additional configuration
+
+- Always-on, default services and features: TLSe
+- Two additional fake baremetal nodes
+- Availability zones for OVN (zone-1 & zone-2)
+- iSCSI service is enabled on all OpenShift nodes.
+
+
+#### iSCSI
+
+It is assumed *iSCSI* services are enabled in all nodes participating in the
```

Comment:
Could this be taken care of by an actual CR that can be applied when kustomize is applied? 

If ci-framework is going to automate this full deploy, it shouldn't be covered only as a manual step.

Reply:
Hello, this change is a draft, not ready for review; it contains copy-paste of what we were told to use as a baseline – please ask the questions there: https://github.com/openstack-k8s-operators/architecture/pull/83

Reply:
No iSCSI in infra (utilized NetApp one) :four_leaf_clover:

Reply:
Ok, my bad, apparently – just for the record, even though NetApp-iSCSI the setup of iscsid and multipathd is needed.

In Zuul job it is handled by DevScripts.

```
cifmw_devscripts_enable_iscsi_on_ocp_nodes: true
cifmw_devscripts_enable_multipath_on_ocp_nodes: true
```